src/utils/constants/tokens.js

```diff
@@ -0,0 +1 @@
+export const mapBoxToken = 'pk.eyJ1Ijoic2pheW1vb24xNSIsImEiOiJjajE2ejhjMXgwM3YxMndtejdpbnlkbWp0In0.HAeIBMg25HSE8zHyNHMccg';
```

Comment:
Add extra blank line at the end... constants like this should be in all caps... MAPBOX_TOKEN